Improve error logs for i/o timeout

Similar to https://github.com/skynetservices/skydns/issues/124?

After the NOTIFY events are send, which have REFUSED (unexpected afaik) and NOTAUTH (expected) as rcodes I get a bunch of i/o timeouts in the form of:
```
2016/11/09 11:20:29 [ERROR] read udp 10.0.1.91:54897->0.0.0.0:53: i/o timeout
2016/11/09 11:20:29 [ERROR] read udp 10.0.1.91:56370->0.0.0.0:53: i/o timeout
```

Some additional info:
I have 6 transfer to statements. 3 for one provider, 2 for another and 1 for AXFR being open from all IPs (firewall prevents abuse for now).

The 3 IPs from provider A are not notified and no i/o timeouts occur on these IPs.
The 2 IPs from provider B are not notified and i/o timeouts occur on these IPs.

--- a/middleware/file/notify.go
+++ b/middleware/file/notify.go
@@ -71,7 +71,7 @@ func notifyAddr(c *dns.Client, m *dns.Msg, s string) (err error) {
 		}
 	}
 	if err != nil {
-		return err
+		return fmt.Errorf("Notify for zone %q was not accepted by %q: %q", m.Question[0].Name, s, err)
 	}
 	return fmt.Errorf("Notify for zone %q was not accepted by %q: rcode was %q", m.Question[0].Name, s, rcode.ToString(code))
 }

